src/controller_examples/zookeeper_controller/exec/reconciler.rs

```diff
@@ -214,17 +214,59 @@ pub fn reconcile_core(
         },
         ZookeeperReconcileStep::AfterCreateStatefulSet => {
             let state_prime = ZookeeperReconcileState {
-                reconcile_step: ZookeeperReconcileStep::Done,
+                reconcile_step: ZookeeperReconcileStep::AfterCreateZKNode,
                 ..state
             };
-            return (state_prime, Option::None);
+            let s1 = cluster_size_zk_node_path(zk);
```

Comment:
Let's try to find better names than s1,2,3 :-))
I am totally fine if it takes more lines

Reply:
Sure.